snap/snapcraft.yaml

```diff
@@ -115,6 +135,9 @@ parts:
       usr/share/seabios/vgabios-stdvga.bin: qemu/
       usr/share/seabios/kvmvapic.bin: qemu/
       usr/lib/ipxe/qemu/efi-virtio.rom: qemu/
+    stage:
```

Comment:
Curious why these need to be unstaged?

Reply:
My local snapcraft runs claims 2 parts supply these files but with different contents. Dunno why

Reply:
Ack, ok